platform/src/main/java/org/stellar/anchor/platform/callback/PlatformIntegrationHelper.java

```diff
@@ -76,7 +77,7 @@ public static AnchorException httpError(String responseContent, int responseCode
     } else if (responseCode == HttpStatus.NOT_FOUND.value()) {
       return new NotFoundException(errorMessage);
     }
-    Log.error(errorMessage);
+    Log.errorF("Unsupported status code {} with body {}", responseCode, errorResponse);
```

Comment:
Could the error response ever contain sensitive information? Sometimes logging data as is can be leak sensitive information like PII, etc.

Reply:
Good point! Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/612/commits/bc1280e2f7319cb1d2ad3c522b2ce367285b6390